lib/Common/Core/ConfigParser.cpp

```diff
@@ -347,9 +347,65 @@ void ConfigParser::ParseRegistryKey(HKEY hk, CmdLineArgsParser &parser)
         }
     }
 
+    SetConfigStringFromRegistry(hk, _u("Telemetry"), _u("Discriminator1"), Js::Configuration::Global.flags.TelemetryDiscriminator1);
+    SetConfigStringFromRegistry(hk, _u("Telemetry"), _u("Discriminator2"), Js::Configuration::Global.flags.TelemetryDiscriminator2);
+    SetConfigStringFromRegistry(hk, _u("Telemetry"), _u("RunType"), Js::Configuration::Global.flags.TelemetryRunType);
+
 #endif // _WIN32
 }
 
+#ifdef _WIN32
```

Comment:
I think this should be even more scoped to NTBUILD- the telemetry code probably shouldn't be compiled into ChakraCore since it can't use those APIs?

Reply:
Left the utility functions in for _WIN32, as they're not telemetry-specific.  Updated telemetry settings to only compile in if `#ifdef ENABLE_BASIC_TELEMETRY